mate-panel/panel-addto.c

```diff
@@ -681,7 +695,12 @@ static void panel_addto_make_application_model(PanelAddtoDialog* dialog)
 	if (dialog->filter_application_model != NULL)
 		return;
 
-	store = gtk_tree_store_new(NUMBER_COLUMNS, G_TYPE_STRING, G_TYPE_STRING, G_TYPE_POINTER, G_TYPE_STRING);
+	store = gtk_tree_store_new(NUMBER_COLUMNS,
```

Comment:
Any chance of using a space before a bracket, same like in the rest of the file?
I see that at several places, i don't want to mark every occurrence in your PR, please fix.

Reply:
Looks much better, thanks.